src/Runner.Worker/Handlers/StepHost.cs

```diff
@@ -239,5 +241,27 @@ public async Task<string> DetermineNodeRuntimeVersion(IExecutionContext executio
                                                          cancellationToken: cancellationToken);
             }
         }
+
+        public DictionaryContextData GetExpressionValues(IExecutionContext executionContext)
```

Comment:
Can we move this to an extension method on execution context. It adds a a dependency on the step host (which we don't want for execution context) but its okay for an extension method.

We should be able to l0 test the extension method using these existing tests, and we can call it in Action Runner